phovea_security_store_ldap/main.py

```diff
@@ -6,7 +6,7 @@ def _loop():
 
     log.setLevel(logging.DEBUG)
     store = LDAPStore()
-    username = input('username: ')
+    username = eval(input('username: '))
```

Comment:
Why do we need the `eval()` here? In conjunction with `input()` this is a [security hole](https://stackoverflow.com/a/9384005)!

Reply:
Thanks for making me aware of that. It has been added by the `2to3` library when converting to the new syntax and I have removed it again.

Reply:
The former`raw_input()` (Python 2.x) has been renamed to  `input()` in Python 3.x. `eval()` has been added to provide the functionality that `input()` provided in 2.x version of python but it is not necessary in this case since it would only be necessary if the code in Python 2.x would have been `eval(raw_input())`